test/types/timezone.jl

```diff
@@ -2,7 +2,7 @@ using TimeZones: Class
 
 @testset "istimezone" begin
     # Invalidate the cache to ensure that `istimezone` works for non-loaded time zones.
-    empty!(TimeZones.TIME_ZONE_CACHE)
+    empty!(TimeZones.default_tz_cache())
```

Comment:
Should be safe as the tests aren't threaded
```suggestion
    TimeZones._reset_tz_cache()
```